client/payment-methods-map.js

```diff
@@ -92,4 +93,15 @@ export default {
 		currencies: [ 'EUR' ],
 		stripe_key: 'sofort_payments',
 	},
+	us_bank_account: {
+		id: 'us_bank_account',
+		label: __( 'US Bank Account (ACH)', 'woocommerce-payments' ),
+		description: __(
+			'US Bank Account needs a description.',
+			'woocommerce-payments'
+		),
+		Icon: BankDebitIcon,
+		currencies: [ 'USD' ],
+		stripe_key: 'us_bank_account_ach_payments',
```

Comment:
Can you change `us_bank_account_ach_payments` to `us_bank_account` as it needs to match the capability name?